_posts/2021-01-29-design-system-crazy.md

```diff
@@ -0,0 +1,137 @@
+teams that cater to the other units, providing a solid platform to run their
+services. We have engineering productivity teams that help others with day to
+day tools (CI, testing environment, etc.). We have analysts and UX researchers
+to support the decision-making of other teams with bespoke data. They are all
+"in-house" services teams.
+
+Organizations willingly invest in all of those but can't form one for the Design
+System? What's the problem?
+
+Is it the idea of mixing designers and developers together as one unit? How is
+it different from what we do for feature teams, squads, swarms or whatever name
+you give to multidisciplinary teams, those that work on features?
+
+Is it an org chart issue? Were it's unclear whether such a team reports to the
+VP of Product or to the VP of Engineering? Why would that matter? Aren't they
+both after the same ultimate company goal?
+
+Is it because designers or developers are already understaffed? Well shoot,
+that's precisely why dedicating a team for it: a Design System is there to help
+you do more with less.
+
+Is it because nobody is here to lead this team? What about those who have
+managed to build the current Design System and do their day to day job *at the
+same time*? Aren't they leadership enough?
+
+Is it because you don't believe in investing in such low levels teams for the
+benefit of rest of the organization? Then stop talking about Design Systems,
+infrastructure teams, UX researchers and so on if you're against such an idea. 
+
+The point to all that questioning is: by wanting to produce a Design System
+without updating your organization to support it, you're making your designers
+and developers go crazy.
+
+They have to support their team projects on the roadmap *while* fighting with
+management for some time for Design System tasks. It's never the priority and
+the roadmap has never enough room for it. But they see its benefits, so they
+keep trying, and trying, and trying.
+
+They end up giving 100% for the Design System (because, you know, such a project
+is not an easy task) *while* they're asked to keep giving 100% on their
+regularly-scheduled features, knowing such a pace always ends poorly (yes,
+burnout). Or wry cynicism in the best of cases, go ask Benoit :)
+
+They have to face deep animosity from other colleagues *while* continuing to
+work with them on their features. Faced with such an unsanctioned, "bottom-up"
+project, some will have no issue undermining the Design System. If it's so
+important, a team would have been formed around it, right? Right.
+
+They are left with two very opposite missions: working for their own specific
+team and its particuliar needs *while simultaneously* creating a product, one
+that accommodates the needs of every team. 
+
+They have to work without the proper communication structures. Nobody listen to
+them. Nobody consider their requests. Nobody respects the rules the Design
+System comes with.
```

Comment:
```suggestion
They also lack the proper communication structures. Nobody listens to
them. Nobody considers their requests. Nobody respects the rules the Design
System comes with because they have no authority. Their product has not been sanctioned, it is not materialized by a single, identifiable core team that must be obeyed. It's just a bunch of people being mad at the void.
```